intuita-webview/src/main/Tree/index.tsx

```diff
@@ -0,0 +1,42 @@
+import TreeView from 'react-treeview';
+import { ReactNode, memo, useState } from 'react';
+import { TreeNode } from '../../../../src/components/webview/webviewEvents';
+
+type Props = {
+	depth?: number;
```

Comment:
ideally this shouldn't be optional

Reply:
i've updated it to `!node.children || node.children.length === 0` because ts was complaining on "node.children can be undefined".